vrk-kpa/suomidigi
# SUOPA-714 Group init

To test: 
- `make fresh`
- Log in as admin and go to https://suomidigi.docker.sh/admin/people
   - Choose your favourite user and edit it. Add role **Groups administrator** for the user and save changes.
- Log in as a user with **Groups administrator** role.
- Go to https://suomidigi.docker.sh/admin/group and click on **Add group**
   - Fill in the fields and click save
   - Check that the group has an url with a prefix of something like _https://suomidigi.docker.sh/ryhmat/<the title of the group>_
   - From the group page, click on **Members / Jäsenet**
       - Click on **Add member** and add a user which has no previous global roles. Select Administrator role for the user and click save.
       - Remove your user from the group. (The one you're logged in at the moment)
- With a different browser: Log in as the user which you added to the group
- Go to https://suomidigi.docker.sh/group/1/edit 
   - Check that the user can edit the group page and that the theme is admin theme
- Try add a third user to the group with content producer group role: https://suomidigi.docker.sh/group/1/members and see that the third user can access the group.
   - If the user can access the group, try to add content to the group
   - If you're able to save the content, you can verify the connection to the group by going to group page https://suomidigi.docker.sh/group/1. The content you've created should be listed on the group front page.
- Finally, with the group administrator, try to translate the group. Check that the url changes as well.
